### PR TITLE
[INLONG-7413][Audit] Audit get MQ config from Manager

### LINKED ISSUE #7413: [Feature][Audit] the Proxy and Store configure MQ address by the Manager service
### Description

Current situation:
- deploy Audit Proxy and Store services need to configure the MQ cluster address
- the MQ cluster address is registered to the Manager after deployed

Expected to happen:
Audit Proxy and Store services can start successfully without an MQ address, and they could fetch the address from the manager when needed.

### Use case

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #7413 

### Motivation

Get the MQ cluster address from the manager.

### Modifications

1. Fix `httpGet` to `httpPost`
2. Add managerHosts and cluster params in `server.properties`
3. Add open API in the manager

### Verifying this change

企业微信截图_16789503041096

### CODE REVIEW

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java [hunk 1]
```diff
@@ -96,4 +127,66 @@ private List<InsertData> getInsertServiceList() {
...
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return null;
+    }
+
+    private List<MQClusterInfo> getMQConfig(String host, String clusterName, String clusterTag) {
+        HttpPost httpPost = null;
+        Gson gson = new Gson();
```
Comment (healchow) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java:153]: Suggest use a global Gson instance.

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java [hunk 2]
```diff
@@ -96,4 +127,66 @@ private List<InsertData> getInsertServiceList() {
...
+            RemoteConfigJson configJson = gson.fromJson(returnStr, RemoteConfigJson.class);
+            if (configJson.isSuccess() && configJson.getData() != null) {
+                LOG.info("getConfig result: {}", configJson);
+                List<MQClusterInfo> mqClusterInfoList = configJson.getData().getMqClusterList();
+                if (mqClusterInfoList != null && !mqClusterInfoList.isEmpty()) {
+                    return mqClusterInfoList;
+                }
+            }
+        } catch (Exception ex) {
+            LOG.error("exception caught", ex);
```
Comment (healchow) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java:183]: Please add more useful info in error log.

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java [hunk 3]
```diff
@@ -96,4 +127,66 @@ private List<InsertData> getInsertServiceList() {
...
+            StringEntity stringEntity = new StringEntity(gson.toJson(request));
+            stringEntity.setContentType("application/json");
+            httpPost.setEntity(stringEntity);
+
+            // request with post
+            LOG.info("start to request {} to get config info with params {}", url, request);
+            CloseableHttpResponse response = httpClient.execute(httpPost);
+            String returnStr = EntityUtils.toString(response.getEntity());
+            // get groupId <-> topic and m value.
+
```
Comment (healchow) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java:173]: Unnecessary blank line here.

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java [hunk 4]
```diff
@@ -96,4 +127,66 @@ private List<InsertData> getInsertServiceList() {
...
+            throw new RuntimeException(e);
+        }
+        return null;
+    }
+
+    private List<MQClusterInfo> getMQConfig(String host, String clusterName, String clusterTag) {
+        HttpPost httpPost = null;
+        Gson gson = new Gson();
+        try {
+            String url = "http://" + host + "/inlong/manager/openapi/dataproxy/getConfig";
```
Comment (healchow) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java:155]: Extract to a global contant.

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java [hunk 5]
```diff
@@ -96,4 +127,66 @@ private List<InsertData> getInsertServiceList() {
         }
         return insertServiceList;
     }
+
+    private List<MQClusterInfo> getConfigManager() {
```
Comment (healchow) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java:131]: `getConfigManager` -> `getClusterListFromManager` maybe more clearly.

File: inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/file/ConfigManager.java [hunk 6]
```diff
@@ -189,33 +200,42 @@ private void checkLocalFile() {
             }
         }
 
-        private boolean checkWithManager(String host) {
-            HttpGet httpGet = null;
+        private boolean checkWithManager(String host, String clusterName, String clusterTag) {
+            HttpPost httpPost = null;
             try {
-                String url = "http://" + host + "/inlong/manager/openapi/audit/getConfig";
+                String url = "http://" + host + "/inlong/manager/openapi/dataproxy/getConfig";
```
Comment (healchow) [inline inlong-audit/audit-common/src/main/java/org/apache/inlong/audit/file/ConfigManager.java:206]: Suggest extract to a constant, or read from the config file.

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java (hunk 5 above)
Comment (haifxu) [reply inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/AuditMsgConsumerServer.java]: Done.

**Review by healchow — APPROVED** [review]

**Review by doleyzi — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**